log/ratelimit_test.go

```diff
@@ -0,0 +1,45 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package logs_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap/zaptest/observer"
+
+	logs "github.com/elastic/apm-server/log"
+	"github.com/elastic/beats/v7/libbeat/logp"
+)
+
+func TestWithRateLimit(t *testing.T) {
```

Comment:
what is this testing?

Reply:
That the 2nd log record is omitted due to rate limiting. I'll add a comment.

Reply:
Updated, please let me know if it's clearer now.

Reply:
Yeah, is more clear, thanks. 
But is this testing any functionality in apm-server? Doesn't this belong to libbeat?

Reply:
Yes, it's testing that the `logs.WithRateLimit` function does what it's supposed to. This function lives in the apm-server repo, and isn't directly tested anywhere else.

Reply:
oh, I'm blind. ofc, thanks